plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageDatabaseUserInterface.java

```diff
@@ -155,11 +190,24 @@ protected void performDefaults()
 
         notificationsEnabled.setSelection(store.getBoolean(ModelPreferences.NOTIFICATIONS_ENABLED));
         notificationsCloseDelay.setSelection(store.getInt(ModelPreferences.NOTIFICATIONS_CLOSE_DELAY_TIMEOUT));
+        final String string = store.getString(DBeaverPreferences.CLIENT_TIMEZONE);
+        if (string.isEmpty()) {
```

Comment:
NPE because the default value is `null`.

Reply:
Well if null is set it returns an empty string. I thought the same about this.